src/test/java/io/burt/athena/result/StandardResultTest.java

```diff
@@ -142,19 +142,22 @@ void doesNotLoadTheSamePageTwice() throws Exception {
         @Nested
         class WhenLoadingIsInterrupted {
             private Thread runner;
-            private AtomicReference<ResultSetMetaData> executeResult;
+            private AtomicReference<Optional<ResultSetMetaData>> executeResult;
```

Comment:
I think you forgot to commit the import of `Optional`, the tests break because they don't find it.

Reply:
Yes, I noticed when I pushed the first time that it had replaced the imports with a star, and I thought it was only my editor being helpful, so I didn't include that chunk, and then I apparently forgot to rerun the tests afterwards.